doc/pages/faq.asciidoc

```diff
@@ -53,23 +53,47 @@ regular color palette by Kakoune. Fortunately, on recent versions of `tmux`
 (>=2.6), you can work around that by using the following configuration
 settings:
 
-~/.tmux.conf
-```
+.~/.tmux.conf
+----
 set -g default-terminal "tmux-256color"
 set -ga terminal-overrides ",*col*:Tc"
-```
+----
 
 Note: `\*col*` is a regular expression that matches your terminal's `$TERM`
 value, modify it if necessary, e.g. `xterm-termite`.
 
-Next, run the following command to create a local `terminfo` override:
+Next, run the following command to create a local `terminfo` override.  If
+your system has a modern ncurses installed, this will be sufficient (see
+notes below if you have an older ncurses or are runing Mac OS X):
 
-```
-$ tic /path/to/kakoune/contrib/tmux-256color.terminfo
-```
+----
+$ tic /path/to/kakoune/contrib/tmux-256color.ncurses61.terminfo
+----
 
 Finally, quit all existing sessions (`tmux kill-server`), and restart `tmux`.
 
+=== Older ncurses
+
+If the above command fails with "signal 11", you likely have a system
+curses which is too old.  In this case, you can get by with the following
+(although a few features will be missing):
+
+----
+$ tic /path/to/kakoune/contrib/tmux-256color.ncurses57.terminfo
+----
+
+=== Mac OS
+
+Mac OS's system ncurses is too old, but Kakoune will automatically use a
+version of ncurses installed with Homebrew, though we need to configure both
+versions of ncurses to support the `tmux-256color` terminal type:
+
+----
+$ brew install ncurses
+$ tic /path/to/kakoune/contrib/tmux-256color.ncurses57.terminfo
+$ /usr/local/opt/ncurses/bin/tic /path/to/kakoune/contrib/tmux-256color.ncurses61.terminfo
```

Comment:
Are you sure it's necessary to run both? I worry about a sequence of events like:

- read system-wide `screen-256color` entry
- add modifications
- write `~/.terminfo/t/tmux-256color`
- read `~/.terminfo/t/tmux-256color`
- add modifications that are already there, so effectively no change
- overwrite `~/.terminfo/t/tmux-256color`

If that's how you've configured your system, it may be worth running `/usr/local/opt/ncurses/bin/infocmp tmux-256color` and see if it contains the tmux-specific entries like `kLFT` and `kRIT`.